Add new parameter css-validate to cloner

viskey98 opened the pull request:
This will create a new parameter `css-validate` to set whether css validation is required or not.
Fixes issue #113

afeena commented on clone.py:
```diff
@@ -165,8 +165,11 @@ def _make_filename(self, url):
-                    css = cssutils.parseString(data)
+                if content_type == 'text/css':                       
+                    if self.css_validate == "False":
+                        css = cssutils.parseString(data, validate=False)
```
Maybe it's better to write:
```cssutils.parseString(data, validate=self.validate)```
In this case you don't need this if/else statement

viskey98 commented:
@afeena Keeping your suggestion in mind.
As the argparse gets input as string (even if we use `type=bool`, any input will validate to True as it is direct string to bool conversion). 
According to me there are two solutions for that:
1. Create a custom function `str_to_bool()` (in a desired way) and in argparse use `type=str_to_bool` .
2. Or else we have to take input as a string in argparse (using `type=str, default="True"`). And have to use an if-else statement before passing the parameter to the Cloner class or after. 
Now, I have changed the code and implemented the first one, please suggest any changes if required :)

afeena commented:
@viskey98 good work :)

viskey98 commented:
@afeena Thank you :)
